### PR TITLE
accounts_db: demote spammy log message

### PR DESCRIPTION
**Pull request by t-nelson**
#### Problem

spammy log message

#### Summary of Changes

demote it

### CODE REVIEW

File: runtime/src/accounts_db.rs [hunk 1]
```diff
@@ -8990,7 +8990,7 @@ impl AccountsDb {
                 if not_sparse || too_small_to_shrink {
                     return 0;
                 }
-                info!(
+                trace!(
```
Comment (t-nelson) [inline runtime/src/accounts_db.rs:8993]: the rest of the logs in this function are `trace`, so that's what i went with.  `debug`'s probably fine to

**Review by yhchiang-sol — APPROVED** [review]
LG.  Trace is a more suitable log level for this.
Make sure to run the formatting before merging 😉 .

**Comment by steviez** [comment]
> LG. Trace is a more suitable log level for this. Make sure to run the formatting before merging 😉 .

Huh, this is the diff `fmt` wants:
```rust
                 trace!(
                     "shrink_stale_slot ({}): not_sparse: {} count: {}/{} byte: {}/{}",
-                    slot, not_sparse, alive_count, stored_count, written_bytes, total_bytes,
+                    slot,
+                    not_sparse,
+                    alive_count,
+                    stored_count,
+                    written_bytes,
+                    total_bytes,
                 );
```
The lines it wants changed were not touched in this PR 🤔

**Comment by t-nelson** [comment]
lmao... `cargo-fmt`, please never change

**Review by steviez — APPROVED** [review]

**Pull request merged**